01_fib/starfallz/main_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestMainFunction(t *testing.T) {
+	t.Run("Test main to return string values of fib(7) with proper formatting", func(t *testing.T) {
+		var buf bytes.Buffer
+		out = &buf
+
+		main()
+
+		expected := strconv.Quote(`1
+1
+2
+3
+5
+8
+13`)
+		actual := strconv.Quote(buf.String())
+
+		if expected != actual {
+			t.Errorf("Unexpected output, expected: %s, actual: %s", expected, actual)
+		}
+	})
+}
+
+func TestFibFunction(t *testing.T) {
+	testCases := []struct {
+		description string
+		input       int
+		expected    []int
+	}{
+		{"Test zero fib to return zero", 0, []int{0}},
```

Comment:
Arguably this shouldn't print anything. Up to you.

Reply:
Ok i will return empty array for this instead of 0